src/state/ducks/userDuck.js

```diff
@@ -77,6 +77,8 @@ const userReducer = (state = userInitialState, action) => {
     case LOGIN_START:
       return { ...state, statusGet: 'pending' };
     case LOGIN_SUCCESS:
+      console.log(action);
```

Comment:
LOGIN_SUCCESS needs to return `{...state, statusGet: "success" };` (and maybe remove the `console.log`

Reply:
oh lol looks like it DOES return that object, just needs to delete `console.log`. So sorry!!!